pkg/cmd/datastore/datastore.go

```diff
@@ -106,6 +107,7 @@ func RegisterDatastoreFlags(cmd *cobra.Command, opts *Config) {
 	cmd.Flags().BoolVar(&opts.ReadOnly, "datastore-readonly", false, "set the service to read-only mode")
 	cmd.Flags().StringSliceVar(&opts.BootstrapFiles, "datastore-bootstrap-files", []string{}, "bootstrap data yaml files to load")
 	cmd.Flags().BoolVar(&opts.BootstrapOverwrite, "datastore-bootstrap-overwrite", false, "overwrite any existing data with bootstrap data")
+	cmd.Flags().DurationVar(&opts.RevisionQuantization, "datastore-bootstrap-timeout", 10*time.Second, "maximum duration before timeout for the bootstrap data to be written")
```

Comment:
```suggestion
	cmd.Flags().DurationVar(&opts. BootstrapTimeout, "datastore-bootstrap-timeout", 10*time.Second, "maximum duration before timeout for the bootstrap data to be written")
```

Reply:
I think it should be possible to now fix https://github.com/authzed/spicedb/issues/741 with this change. We don't have to do it in this PR.